barberia/src/App/pages/covertdate.js

```diff
@@ -0,0 +1,29 @@
+export function convertDate(str) {
+  var mnths = {
+      Jan: "01",
+      Feb: "02",
+      Mar: "03",
+      Apr: "04",
+      May: "05",
+      Jun: "06",
+      Jul: "07",
+      Aug: "08",
+      Sep: "09",
+      Oct: "10",
+      Nov: "11",
+      Dec: "12"
+    },
+    date = str.split(" ");
+  var filterd_date = [date[3], mnths[date[1]], date[2]].join("");
```

Comment:
split `date[1]` `date[2]` `date[3]` into `consts or constants` so we know what each one is

also make filterd_date a `const`

Reply:
fixed